tox.ini

```diff
@@ -126,3 +126,25 @@ commands =
 description = Run the mypy static type checker
 deps = mypy
 commands = mypy {toxinidir}/dkist {posargs}
+
+[testenv:dkist-inventory]
+description = Run the dkist-inventory tests against this version of dkist
+base_python = py311  # dkist-inventory is fussy about python version
+extras = # we don't need dkist test extras
+allowlist_externals = git, rm, bash
+commands =
+    bash -c 'rm -rf dkist-inventory || true'
+    git clone https://bitbucket.org/dkistdc/dkist-inventory.git
+    python -m pip install "./dkist-inventory[test,asdf]"
```

Comment:
Doesn't hurt, but the asdf extra is in the test extra so shouldn't be needed. https://bitbucket.org/dkistdc/dkist-processing-common/src/9fd962944f5c4dca38a330d799138a9af7cb2fcc/pyproject.toml#lines-65